lib/_app/AppBar/AppBar.tsx

```diff
@@ -1,63 +1,108 @@
-import {
-  AppBar as MuiAppBar,
-  Container,
-  IconButton,
-  Typography,
-} from '@mui/material';
-import React, { useState } from 'react';
 import Image from 'next/image';
-import { Icon } from '~/lib/components/Icon';
-import SearchIcon from '~/public/images/icons/search-regular.svg';
-import TimesIcon from '~/public/images/icons/times-regular.svg';
-import s from './AppBar.module.scss';
 import Link from 'next/link';
+import React, { useState } from 'react';
+import { useRouter } from 'next/router';
+import clsx from 'clsx';
 
 export const AppBar = () => {
   const [showSearch, setShowSearch] = useState(false);
+  const router = useRouter();
+  const isHomepage = router.pathname == '/';
 
   return (
-    <MuiAppBar
-      elevation={1}
-      color="transparent"
-      position="static"
-      className={s.AppBar}
-    >
-      <Container maxWidth="md" className={s.container}>
-        <div className={s.logo}>
-          <div style={{ display: showSearch ? 'block' : 'none' }}>
-            <div title="Google search results" className="gcse-search" />
-          </div>
-
-          <div style={{ display: showSearch ? 'none' : 'block' }}>
+    <nav className={isHomepage ? 'py-10' : 'py-4'}>
+      <section
+        className={clsx(
+          'px-4 container mx-auto flex gap-4 items-center justify-center w-full h-auto',
+          isHomepage ? 'flex-col' : 'flex-row',
+        )}
+      >
+        {!showSearch && (
```

Comment:
I noticed that you were using logic like `showSearch ? 'hidden' : 'flex'`. In React you can do instead:

```jsx
{!showSearch && <Component />}
```

...so I converted all of the code where you were using 'hidden' class to the React way.

Let me know if you have any concerns. If you don't have any concerns, you can mark this as resolved.

Reply:
When I pulled this, I can't see the search bar input on large screens on the celeb pages, but I can see it in small screens
![image](https://user-images.githubusercontent.com/68635840/165455848-7148e937-634d-49f8-b8d4-703dfc9b0e53.png)

Reply:
I fixed it using the clx https://github.com/hollowverse/hollowverse/pull/69/commits/ef564f0b3b2faa07994acf501710ae41de18d896